example_project_app/plugins.py

```diff
@@ -211,6 +216,24 @@ class ProjectAppPlugin(ProjectModifyPluginMixin, ProjectAppPluginPoint):
             'default': '',
             'description': 'Example json project user setting',
         },
+        'project_callable_setting': {
+            'scope': SODAR_CONSTANTS['APP_SETTING_SCOPE_PROJECT'],
+            'type': 'BOOLEAN',
+            'default': callable_function(),
+            'description': 'Example callable project setting',
```

Comment:
For `PROJECT` and `USER` settings, I also recommend setting the `label` attribute. (for `PROJECT_USER` it is not needed, as those are not displayed in project/user forms)